mscr-ui/src/common/utils/has-permission.tsx

```diff
@@ -10,22 +10,6 @@ import { User } from 'yti-common-ui/interfaces/user.interface';
 
 // Need to specify the acctions permitted for each type of user
 const actions = [
```

Comment:
With this implementation, I guess we don't even need actions. I checked the datamodel-ui version, and it seems they use the actions to check for specifically admin-level actions. Our actions seem to be either creating something or editing something, and you've defined paths to tackle each case. All the logic is based on whether the util is called with an owner and with a target organization. So should we just simplify everything and take out all mentions of actions?

Reply:
Will do in the next iteration. We are really not using the action specific things. But may be we can put it back in future if really needed